dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html

```diff
@@ -118,15 +118,15 @@
         </div>
       </ng-form>
     </che-label-container>
-    <!--Workspace RAM--Temporary not available //TODO add when implemented>
-   <!-- <che-label-container che-label-name="RAM">
```

Comment:
if you uncomment this section, then provide the valid RAM value retrieving. Check "memoryLimitBytes" machine's attribute.